### PR TITLE
Fix svg icons

### PR DESCRIPTION
**Pull request by bitwiseman**
Some of the GitHub icons were not transcribed correctly.
Also split onto multiple lines for easier viewing.

# Description

A brief summary describing the changes in this pull request. See 
[JENKINS-66148](https://issues.jenkins-ci.org/browse/JENKINS-66148) for further information. 

# Submitter checklist
- [ ] Link to JIRA ticket in description, if appropriate.
- [ ] Change is code complete and matches issue description
- [ ] Automated tests have been added to exercise the changes
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.

# Reviewer checklist
- [ ] Run the changes and verify that the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

# Documentation changes
- [ ] Link to jenkins.io PR, or an explanation for why no doc changes are needed

# Users/aliases to notify

### CODE REVIEW

File: src/main/webapp/images/svgs/sprite-github.svg [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
         <path fill-rule="evenodd" d="M8 0C3.58 0 0 3.58 0 8c0 3.54 2.29 6.53 5.47 7.59.4.07.55-.17.55-.38 0-.19-.01-.82-.01-1.49-2.01.37-2.53-.49-2.69-.94-.09-.23-.48-.94-.82-1.13-.28-.15-.68-.52-.01-.53.63-.01 1.08.58 1.23.82.72 1.21 1.87.87 2.33.66.07-.52.28-.87.51-1.07-1.78-.2-3.64-.89-3.64-3.95 0-.87.31-1.59.82-2.15-.08-.2-.36-1.02.08-2.12 0 0 .67-.21 2.2.82.64-.18 1.32-.27 2-.27.68 0 1.36.09 2 .27 1.53-1.04 2.2-.82 2.2-.82.44 1.1.16 1.92.08 2.12.51.56.82 1.27.82 2.15 0 3.07-1.87 3.75-3.65 3.95.29.25.54.73.54 1.48 0 1.07-.01 1.93-.01 2.2 0 .21.15.46.55.38A8.013 8.013 0 0016 8c0-4.42-3.58-8-8-8z"/>
     </symbol>
     <symbol id="github-repo" viewBox="0 0 24 24">
         <path fill-rule="evenodd" d="M3 2.75A2.75 2.75 0 015.75 0h14.5a.75.75 0 01.75.75v20.5a.75.75 0 01-.75.75h-6a.75.75 0 010-1.5h5.25v-4H6A1.5 1.5 0 004.5 18v.75c0 .716.43 1.334 1.05 1.605a.75.75 0 01-.6 1.374A3.25 3.25 0 013 18.75v-16zM19.5 1.5V15H6c-.546 0-1.059.146-1.5.401V2.75c0-.69.56-1.25 1.25-1.25H19.5z"/>
+        <path d="M7 18.25a.25.25 0 01.25-.25h5a.25.25 0 01.25.25v5.01a.25.25 0 01-.397.201l-2.206-1.604a.25.25 0 00-.294 0L7.397 23.46a.25.25 0 01-.397-.2v-5.01z"/>
```
Comment (bitwiseman) [inline src/main/webapp/images/svgs/sprite-github.svg:8]: This element was missing.

File: src/main/webapp/images/svgs/sprite-github.svg [hunk 2]
```diff
@@ -1,12 +1,15 @@
 <!-- Sourced from https://github.com/primer/octicons -->
 <svg xmlns="http://www.w3.org/2000/svg">
-    <symbol id="github-logo" viewBox="0 0 24 24">
+    <symbol id="github-logo" viewBox="0 0 16 16">
```
Comment (bitwiseman) [inline src/main/webapp/images/svgs/sprite-github.svg:3]: This attribute didn't match the path element below, making the icon smaller than it should be.

File: src/main/webapp/images/svgs/sprite-github.svg [hunk 3]
```diff
@@ -1,12 +1,15 @@
...
+        <path fill-rule="evenodd" d="M5.75 21a1.75 1.75 0 110-3.5 1.75 1.75 0 010 3.5zM2.5 19.25a3.25 3.25 0 106.5 0 3.25 3.25 0 00-6.5 0zM5.75 6.5a1.75 1.75 0 110-3.5 1.75 1.75 0 010 3.5zM2.5 4.75a3.25 3.25 0 106.5 0 3.25 3.25 0 00-6.5 0zM18.25 6.5a1.75 1.75 0 110-3.5 1.75 1.75 0 010 3.5zM15 4.75a3.25 3.25 0 106.5 0 3.25 3.25 0 00-6.5 0z"/>
+        <path fill-rule="evenodd" d="M5.75 16.75A.75.75 0 006.5 16V8A.75.75 0 005 8v8c0 .414.336.75.75.75z"/>
+        <path fill-rule="evenodd" d="M17.5 8.75v-1H19v1a3.75 3.75 0 01-3.75 3.75h-7a1.75 1.75 0 00-1.75 1.75H5A3.25 3.25 0 018.25 11h7a2.25 2.25 0 002.25-2.25z"/>
```
Comment (bitwiseman) [inline src/main/webapp/images/svgs/sprite-github.svg:13]: Formatting only - split to multiple lines.

**Review by jtnord — APPROVED** [review]

**Review by timja — APPROVED** [review]
nice thanks!

https://issues.jenkins.io/browse/JENKINS-66100 looks to be a duplicate as well?

**Comment by bitwiseman** [comment]
@timja 
There's additional changes needed as follow-on to this PR. We need a fix in the folders plugin for this: 

https://github.com/jenkinsci/cloudbees-folder-plugin/blob/9130027fac8ae23f3637e45f731679c196437040/src/main/resources/com/cloudbees/hudson/plugins/folder/AbstractFolder/view-index-top.jelly#L27-L30

The `img` element doesn't seem to understand the `#github-repo` part of the URL.  I'm digging back into https://github.com/jenkinsci/jenkins/pull/5065 to try to understand what changes would be needed.

**Review by alecharp — APPROVED** [review]

**Pull request merged**